src/api/app/views/webui/main/_contextual_actions.html.haml

```diff
@@ -0,0 +1,18 @@
+- content_for :contextual_actions do
```

Comment:
I don't think they are in context of the main page only. Those should just always be there no?

Reply:
Thinking about it... How about

- move `Configuration` to the avatar menu, it's a place not an action
- show `New Image` only in the `Project` actions (would probably imply to move away from branching later)
- show `Create Status Message` in the main page actions (if you're an Admin)
- move `Create Project` to the `Configuration` actions. Creating top level projects is only possible for admins anyway...

Reply:
>     * move `Configuration` to the avatar menu, it's a place not an action

I agree.

>     * show `New Image` only in the `Project` actions (would probably imply to move away from branching images later)

Alright, this will be done in a following PR for project#show (it's the next in line).

>     * show `Create Status Message` in the main page actions

This is already the case, so nothing has to change for this.

>     * move `Create Project` to the `Configuration` actions. Creating top level projects is only possible for admins anyway...

Users can create projects which aren't top level as its explained with this message above the form on the page:

`All users have a homes project by default. Yours is home:user_1, and you can also create sub projects inside. New projects outside of your home project can get requested via email.`

With this in mind, I would keep `Create Project` on the main page actions.

Reply:
sorry for the back and forth...

Reply:
No worries, please have another look.